### ISSUE TITLE
Resize tool is bad for pixel graphics

### ISSUE THREAD

**Issue by BonzerKitten7** [Scirra/Construct-3-bugs#1437]
## Problem description

As Ashley probably already knows, I'm currently re-scaling my whole project by 4, it includes all the 200 sprites that I'm using, but the problem is that all my sprites have pixel art graphics, so in order to be able to scale all of them i need to use a nearest neighbor algorithm, but Construct 3 only offers bicubic algorithm (i guess).

https://i.imgur.com/GSOBGTc.png?1

I guess the best solution is to add a dropdown menu to choose which method you want use, like in Photoshop.

https://i.imgur.com/DbBdYu3.png?1

## Attach a .c3p

[ResizeProblem.zip](https://github.com/Scirra/Construct-3-bugs/files/1853175/ResizeProblem.zip)

## Steps to reproduce

1. Create a sprite object.
2. Add pixel art graphics to the sprite that you just created.
3. Resize your sprite.

## Observed result

The image gets blurry.

## Expected result

A not blurry image.

## Affected browsers

* Chrome: yes

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by Scoremonger** [comment]
I noticed this in my project too. I have sampling set to "Point", but the resized sprite is extremely blurry inside the animation editor. But this is only after the initial resize, and only in the animation editor. So it seems there is a bug only with the initial presentation of the resized sprite - the final result isn't as blurry as it appears. If you resize, then close the animation editor, does the resized sprite look as blurry in the layout? 

I took some screenshots of a random pixel art guy from the web. The initial resize is crazy blurry. Then in the layout it looks OK. Then after reopening it in the animation editor it still looks OK.
![1-initialresize](https://user-images.githubusercontent.com/28039905/37993994-01c8a9ec-31c5-11e8-925b-c83c3d108ee8.jpg)
![2-samespriteinlayout](https://user-images.githubusercontent.com/28039905/37993995-01e0a4ca-31c5-11e8-92b1-a54894eaae45.jpg)
![3-samespritereopenedinanimationeditor](https://user-images.githubusercontent.com/28039905/37993996-01f42f9a-31c5-11e8-82eb-b59bee7d0d2c.jpg)

**Comment by joelmayer87** [comment]
This is because there's no point or nearest neighbour interpolation option in the image editor. Don't know if that's really a bug or just a missing feature but i'd love to see the option available as well since it would make some things a lot easier for people using pixel art.

**Comment by AshleyScirra** [comment]
FWIW you can scale your entire project without having to stretch the artwork itself. (This is still a useful feature though.)

**Comment by DiegoScirra** [comment]
I added a new **"Smooth"** checkbox to the resize dialog. Turning it off seems to be producing better results when working with small pixel art images.

Check it out in r95!

**Comment by BonzerKitten7** [comment]
Thank you, Diego! It's much appreciated. :)

**Comment by AshleyScirra** [comment]
r95 is now out, can you confirm the release solves this?

**Comment by BonzerKitten7** [comment]
Sure, I'll check that out! I'm currently submitting a new issue about tiled backgrounds inside global layers.
